src/services/createRouter.spec-d.ts

```diff
@@ -0,0 +1,100 @@
+import { createRoute } from '@/services/createRoute'
+import { createRouter } from '@/services/createRouter'
+import { component } from '@/utilities/testHelpers'
+import { describe, test, expectTypeOf } from 'vitest'
+import { AddRouterAfterRouteHook, AddRouterBeforeRouteHook } from '@/types/router'
+import { createRouterPlugin } from './createRouterPlugin'
+import { CallbackContextAbort } from './createCallbackContext'
+
+describe('createRouter', () => {
```

Comment:
nit: this test doesn't describe its test case very well

Reply:
Good catch, updated